### ISSUE TITLE
Sharing: Eventbrite connection can't be disconnected

### ISSUE THREAD

**Issue by adelineyaw** [Automattic/wp-calypso#23285]
I had a user in chat and they can't seem to disconnect their Eventbrite account (within Sharing > Other Connections)

Once you click on 'disconnect', you can see the 'successfully disconnected' message:

<img width="534" alt="screen shot 2018-03-14 at 7 21 57 am" src="https://user-images.githubusercontent.com/7424885/37371907-1f44cdda-2765-11e8-9861-a8babb7f4d44.png">

However once you navigate away and come back to the Publicize page, it still appears connected to 'Pastor Michael Johnson' as seen below:

<img width="765" alt="screen shot 2018-03-14 at 8 57 43 am" src="https://user-images.githubusercontent.com/7424885/37372330-78586868-2766-11e8-9afe-25e884c3405a.png">

I've tried to disconnect their Eventbrite account via SU and SSP and get the same results (as above).

URL: [victorscott.org](https://victorscott.org)
Chat ID: #2051054-hc
Follow up ticket ID: #1016065-zen

**Comment by rachelmcr** [comment]
I reproduced this issue on a test site with the following steps:

1. Starting at URL: https://wordpress.com/sharing
2. Select a site from the site picker.
3. Next to "Eventbrite" select "Connect" and follow the instructions in the dialog to log in to Eventbrite and approve the connection.
4. Next to "Eventbrite" select "Disconnect."

Results: The success message appears but the connection is still there.

No console errors. The network request to the `settings` API endpoint sends the request payload `{eventbrite_api_token: null}` and receives this response:

```
{"code":200,"headers":[{"name":"Content-Type","value":"application\/json"}],"body":{"updated":[]}}
```

**Comment by benchilcote** [comment]
I ran into this issue as well when adding connecting a test Eventbrite account to a user's WordPress.com account to test another issue (with the user's permission). I have not been able to remove myself. I followed the same process as the others and achieved the same result.

Here is an animation that starts just after I disconnected and received the "success" message, and continues all the way until I refresh the page and show my name is still there.

![eventbrite disconnect](https://user-images.githubusercontent.com/11636210/40733574-f6424242-6403-11e8-888c-0076ed37f9b2.gif)
